client/packages/system/src/Patient/VaccinationCards/DetailView/DetailView.tsx

```diff
@@ -12,30 +11,63 @@ import {
   NothingHere,
   LocaleKey,
   useFormatDateTime,
+  useRowStyle,
+  useTheme,
+  StatusCell,
 } from '@openmsupply-client/common';
 import {
   usePatientVaccineCard,
   VaxCardData,
 } from '../../api/hooks/usePatientVaccineCard';
 
+const useStyleRowsByStatus = (rows: DoseRowData[]) => {
+  const { setRowStyles } = useRowStyle();
+  const theme = useTheme();
+
+  useEffect(() => {
+    if (!rows) return;
+
+    setRowStyles(
+      rows.filter(row => !!row.dateGiven).map(row => row.id),
+      {
+        backgroundColor: `${theme.palette.background.success} !important`,
+      }
+    );
+  }, [rows]);
+};
+
 export const VaccinationCardComponent: FC = () => {
   const t = useTranslation('dispensary');
   const { patientId = '', programEnrolmentId = '' } = useParams();
   const { setCustomBreadcrumbs } = useBreadcrumbs();
   const { localisedDate } = useFormatDateTime();
+  const theme = useTheme();
 
   const {
     query: { data, isLoading },
   } = usePatientVaccineCard(patientId, programEnrolmentId);
 
-  const tableData = buildTableData(data);
+  const tableData = useMemo(() => buildTableData(data), [data]);
+
+  useStyleRowsByStatus(tableData);
+
+  useEffect(() => {
+    if (data) {
+      setCustomBreadcrumbs({
+        1: data?.vaccineCardItems?.patient?.name ?? '',
+        2: t('label.vaccination-card'),
+        3: data?.vaccineCardItems?.programName,
+      });
+    }
+  }, [data]);
```

Comment:
Moved, not changed.

Reply:
Can also be in the next PR, but would be good to refactor the table to a separate component from the detail view, so we can reuse in the encounter, without the breadcrumb bits etc 🚀